bears/generate_package.py

```diff
@@ -87,15 +98,48 @@ def create_upload_parser():
     :return: Parser arguments.
     """
     parser = argparse.ArgumentParser(
-        description='Generates PyPi packages from bears.')
+        description='Generates PyPI packages from bears.')
     parser.add_argument('-r', '--register',
-                        help='Register the packages on PyPi',
+                        help='Register the packages on PyPI',
                         action='store_true')
-    parser.add_argument('-u', '--upload', help='Upload the packages on PyPi',
+    parser.add_argument('-u', '--upload', help='Upload the packages on PyPI',
                         action='store_true')
+    parser.add_argument('-c', '--conda', help='Create a conda package',
+                        metavar='dir')
     return parser
 
 
+def get_bear_glob(path):
+    """
+    Returns a list of filenames depending on the type of package that is to be
+    created.
+
+    :param path: Path from which to fetch the Bear if a conda package is to be
+                 created.
+    :return:     Returns a list with one filename for a conda package or a
+                 list with multiple filenames for the PyPI packages.
+    """
+    return (list(glob(os.path.join(path, '*Bear.py')))
+            if path else sorted(set(glob('bears/**/*Bear.py'))))
+
+
+def fetch_url(path):
+    """
+    Checks the ``.git`` directory in ``path`` and fetches the URL of the repo.
+
+    :param path: Path in which to look for a ``.git`` directory.
+    :return:     Returns the url fetched or ``False`` if the URL could not be
+                 fetched.
+    """
+    config = os.path.join(path, '.git', 'config')
+    try:
+        with open(config, 'r') as f:
+            match = re.search(r'\s*url = (?P<url>.+)', f.read())
+            return match.group('url') if match.group('url') else None
+    except FileNotFoundError:
```

Comment:
There can also be OSError/IOError is the file exists but is not readable right ?